Quetoo.vs15/src/config.h

```diff
@@ -34,6 +34,10 @@
 		g_list_foreach(list, (GFunc)func, NULL); \
 		g_list_free(list);
 
+#define g_slist_free_full(list, func)	\
+		g_slist_foreach(list, (GFunc)func, NULL); \
+		g_slist_free(list);
+

```

Comment:
As a low-priority item, it would be great to either move this into a glib-specific include, or really, just snag an updated glib.dll and glib.h that includes this stuff. It has to exist, no? Tons of GTK+ software is available on Windows..